cloud-nio/cloud-nio-spi/src/main/scala/cloud/nio/spi/CloudNioRetry.scala

```diff
@@ -4,17 +4,18 @@ import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 
 import scala.concurrent.duration._
+import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 class CloudNioRetry(config: Config) {
 
-  def defaultMaxRetries: Int = config.getAs[Int]("default-max-retries").getOrElse(10)
+  def defaultMaxRetries: Int = config.getAs[Int]("default-max-retries").getOrElse(3)
```

Comment:
Again, no strong empirical evidence, but I believe this "10" was copied from the behavior of the built-in Papi V1 localizer / delocalizer that retried that number of times.

This does appear to be touching an SPI (thus a common interface) and not an implementation, but still LGTM

Reply:
oh excellent point, I think I may revert this bit then

Reply:
Just agreeing that we shouldn't change this if it affects things more universally, especially if we're not sure if the potential benefit is worth the potential risk of collateral effects.